tools/keycloak/sync/appsettings.test.json

```diff
@@ -24,6 +24,6 @@
   "RetryAttempts": 3,
   "AbortAfterFailure": 1,
   "Api": {
-    "Uri": "https://pims-test.apps.silver.devops.gov.bc.ca/api"
+    "Uri": "https://pims-app-test-3cd915-dev.apps.silver.devops.gov.bc.ca/api"
```

Comment:
Wish they would make friendly URLS.

Reply:
we do have friendly urls:
* https://dev-pims.th.gov.bc.ca/api
* https://tst-pims.th.gov.bc.ca/api
* https://uat-pims.th.gov.bc.ca/api

Reply:
that's true, could update to use those instead.

Reply:
yes please